2019-10-14 08:33:43 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth token' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(66)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(634): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(679): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getUsers(NULL)
#6 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(844): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getAllUsers()
#7 /Applications/MAMP/htdocs/zoho/index.php(21): zcrmsdk\crm\setup\org\ZCRMOrganization->getAllUsers()
#8 {main}
2019-10-14 08:35:27 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth token' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(613): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(834): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getUser('217620000000184...')
#6 /Applications/MAMP/htdocs/zoho/index.php(21): zcrmsdk\crm\setup\org\ZCRMOrganization->getUser('217620000000184...')
#7 {main}
2019-10-14 08:39:21 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth token' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(613): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(834): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getUser('217620000000184...')
#6 /Applications/MAMP/htdocs/zoho/index.php(35): zcrmsdk\crm\setup\org\ZCRMOrganization->getUser('217620000000184...')
#7 /Applications/MAMP/htdocs/zoho/index.php(100): Org->getUser()
#8 {main}
2019-10-14 08:41:34 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth token' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(613): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(834): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getUser('217620000000184...')
#6 /Applications/MAMP/htdocs/zoho/index.php(21): zcrmsdk\crm\setup\org\ZCRMOrganization->getUser('217620000000184...')
#7 {main}
2019-10-14 08:42:14 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth token' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(613): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(834): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getUser('217620000000184...')
#6 /Applications/MAMP/htdocs/zoho/index.php(22): zcrmsdk\crm\setup\org\ZCRMOrganization->getUser('217620000000184...')
#7 {main}
2019-10-14 08:47:32 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'INVALID_DATA-The given id seems to be invalid.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(88)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(613): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(834): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getUser('217620000000184...')
#6 /Applications/MAMP/htdocs/zoho/index.php(65): zcrmsdk\crm\setup\org\ZCRMOrganization->getUser('217620000000184...')
#7 {main}
2019-10-14 08:48:45 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'INVALID_DATA-The given id seems to be invalid.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(88)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(613): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(834): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getUser('217620000000184...')
#6 /Applications/MAMP/htdocs/zoho/index.php(65): zcrmsdk\crm\setup\org\ZCRMOrganization->getUser('217620000000184...')
#7 {main}
2019-10-14 08:50:41 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'INVALID_DATA-The given id seems to be invalid.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(88)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(613): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(834): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getUser('217620000000184...')
#6 /Applications/MAMP/htdocs/zoho/index.php(62): zcrmsdk\crm\setup\org\ZCRMOrganization->getUser('217620000000184...')
#7 {main}
2019-10-14 09:01:02 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'INVALID_DATA-The given id seems to be invalid.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(88)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(613): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(834): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getUser('217620000000184...')
#6 /Applications/MAMP/htdocs/zoho/index.php(68): zcrmsdk\crm\setup\org\ZCRMOrganization->getUser('217620000000184...')
#7 {main}
2019-10-14 09:07:29 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'unable to process your request. please verify whether you have entered proper method name, parameter and parameter values.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 /Applications/MAMP/htdocs/zoho/index.php(66): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 {main}
2019-10-14 09:07:56 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'unable to process your request. please verify whether you have entered proper method name, parameter and parameter values.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 /Applications/MAMP/htdocs/zoho/index.php(66): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 {main}
2019-10-14 09:09:51 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth scope to access this URL' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 /Applications/MAMP/htdocs/zoho/index.php(67): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 {main}
2019-10-14 09:09:57 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'unable to process your request. please verify whether you have entered proper method name, parameter and parameter values.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 /Applications/MAMP/htdocs/zoho/index.php(66): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 {main}
2019-10-14 09:15:23 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'unable to process your request. please verify whether you have entered proper method name, parameter and parameter values.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(66)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(528): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(944): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getAllProfiles()
#6 /Applications/MAMP/htdocs/zoho/index.php(65): zcrmsdk\crm\setup\org\ZCRMOrganization->getAllProfiles()
#7 {main}
2019-10-14 09:16:26 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'unable to process your request. please verify whether you have entered proper method name, parameter and parameter values.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(66)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(528): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(944): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getAllProfiles()
#6 /Applications/MAMP/htdocs/zoho/index.php(65): zcrmsdk\crm\setup\org\ZCRMOrganization->getAllProfiles()
#7 {main}
2019-10-14 09:57:23 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'unable to process your request. please verify whether you have entered proper method name, parameter and parameter values.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(66)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(528): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/setup/org/ZCRMOrganization.php(944): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getAllProfiles()
#6 /Applications/MAMP/htdocs/zoho/index.php(65): zcrmsdk\crm\setup\org\ZCRMOrganization->getAllProfiles()
#7 {main}
2019-10-14 10:07:44 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth scope to access this URL' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(66)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/MassEntityAPIHandler.php(296): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1086): zcrmsdk\crm\api\handler\MassEntityAPIHandler->getRecords(NULL, NULL, NULL, 1, 100, NULL)
#6 /Applications/MAMP/htdocs/zoho/index.php(70): zcrmsdk\crm\crud\ZCRMModule->getRecords(NULL, NULL, NULL, 1, 100, NULL)
#7 {main}
2019-10-14 10:17:14 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'INVALID_DATA-The given id seems to be invalid.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(88)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 /Applications/MAMP/htdocs/zoho/index.php(70): zcrmsdk\crm\crud\ZCRMModule->getRecord('217620000000184...')
#7 {main}
2019-10-14 10:17:22 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'INVALID_DATA-The given id seems to be invalid.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(88)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 /Applications/MAMP/htdocs/zoho/index.php(70): zcrmsdk\crm\crud\ZCRMModule->getRecord('217620000000184...')
#7 {main}
2019-10-14 10:17:30 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'INVALID_DATA-The given id seems to be invalid.' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(88)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204, NULL)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 /Applications/MAMP/htdocs/zoho/index.php(70): zcrmsdk\crm\crud\ZCRMModule->getRecord('217620000000184...')
#7 {main}
2019-10-14 12:45:13 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(61)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/MassEntityAPIHandler.php(337): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1125): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords('krismarrierrr@g...', 1, 200, 'email')
#6 /Applications/MAMP/htdocs/zoho/functions.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByEmail('krismarrierrr@g...', 1, 200)
#7 /Applications/MAMP/htdocs/zoho/index.php(7): get_contact_by_email('krismarrierrr@g...')
#8 {main}
2019-10-14 12:46:44 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(61)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/MassEntityAPIHandler.php(337): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1125): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords('krismarrierrr@g...', 1, 200, 'email')
#6 /Applications/MAMP/htdocs/zoho/functions.php(36): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByEmail('krismarrierrr@g...', 1, 200)
#7 /Applications/MAMP/htdocs/zoho/index.php(7): get_contact_by_email('krismarrierrr@g...')
#8 {main}
2019-10-14 12:47:13 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(61)
Trace::#0 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/MassEntityAPIHandler.php(337): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 /Applications/MAMP/htdocs/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1125): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords('krismarrierrr@g...', 1, 200, 'email')
#6 /Applications/MAMP/htdocs/zoho/functions.php(37): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByEmail('krismarrierrr@g...', 1, 200)
#7 /Applications/MAMP/htdocs/zoho/index.php(7): get_contact_by_email('krismarrierrr@g...')
#8 {main}
2019-11-07 15:29:19 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(38): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(8): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 15:29:31 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(38): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(9): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 15:30:24 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(38): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(12): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 15:31:07 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(38): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(12): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 15:39:10 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(38): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(7): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 15:39:29 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(38): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(7): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 15:40:16 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(38): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(7): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 15:46:26 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(38): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(7): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 16:00:34 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(46): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 16:01:02 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(46): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 16:11:00 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(46): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 16:18:03 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(46): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 16:47:37 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(39): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 16:59:16 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth scope to access this URL' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(39): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 /Users/nicola/Desktop/Web/Apache/zoho/index.php(13): Zoho_Connection_By_Gatto->check_connection()
#8 {main}
2019-11-07 17:00:36 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth scope to access this URL' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(39): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#8 {main}
2019-11-07 17:01:12 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth scope to access this URL' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(39): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#8 {main}
2019-11-07 17:01:40 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth scope to access this URL' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(39): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#8 {main}
2019-11-07 17:03:35 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth scope to access this URL' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(39): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#8 {main}
2019-11-07 17:04:19 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(39): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-07 17:04:38 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth token' in /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /Users/nicola/Desktop/Web/Apache/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 /Users/nicola/Desktop/Web/Apache/zoho/zoho-connection.php(39): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 /Users/nicola/Desktop/Web/Apache/zoho/index.php(10): Zoho_Connection_By_Gatto->check_connection()
#8 {main}
2019-11-11 07:52:39 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(39): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/index.php(18): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-11 07:53:10 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/utility/ZCRMConfigUtil.php(93)
Trace::#0 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/setup/restclient/ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(39): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/index.php(18): Zoho_Connection_By_Gatto->check_connection()
#6 {main}
2019-11-15 11:32:16 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404, NULL)
#3 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(117): zcrmsdk\crm\crud\ZCRMModule->getRecord('null')
#7 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(123): Zoho_Connection_By_Gatto->get_contact('null')
#8 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/index.php(40): Zoho_Connection_By_Gatto->get_contact_field_value('null', 'Coordinate_Tett...')
#9 {main}
2019-11-15 11:39:01 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404, NULL)
#3 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(117): zcrmsdk\crm\crud\ZCRMModule->getRecord('null')
#7 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(123): Zoho_Connection_By_Gatto->get_contact('null')
#8 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/index.php(41): Zoho_Connection_By_Gatto->get_contact_field_value('null', 'Coordinate_Tett...')
#9 {main}
2019-11-15 11:39:07 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404, NULL)
#3 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(117): zcrmsdk\crm\crud\ZCRMModule->getRecord('null')
#7 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(123): Zoho_Connection_By_Gatto->get_contact('null')
#8 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/index.php(41): Zoho_Connection_By_Gatto->get_contact_field_value('null', 'Coordinate_Tett...')
#9 {main}
2019-11-15 11:39:08 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404, NULL)
#3 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(117): zcrmsdk\crm\crud\ZCRMModule->getRecord('null')
#7 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(123): Zoho_Connection_By_Gatto->get_contact('null')
#8 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/index.php(41): Zoho_Connection_By_Gatto->get_contact_field_value('null', 'Coordinate_Tett...')
#9 {main}
2019-11-15 11:39:14 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404, NULL)
#3 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(117): zcrmsdk\crm\crud\ZCRMModule->getRecord('null')
#7 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(123): Zoho_Connection_By_Gatto->get_contact('null')
#8 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/index.php(41): Zoho_Connection_By_Gatto->get_contact_field_value('null', 'Coordinate_Tett...')
#9 {main}
2019-11-15 11:39:52 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(93)
Trace::#0 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/response/APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404, NULL)
#3 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/api/handler/EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/vendor/zohocrm/php-sdk/src/crm/crud/ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(117): zcrmsdk\crm\crud\ZCRMModule->getRecord('null')
#7 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/zoho-connection.php(123): Zoho_Connection_By_Gatto->get_contact('null')
#8 /home/customer/www/sunprime.it/public_html/traccia-il-tuo-tetto/php/zoho/index.php(41): Zoho_Connection_By_Gatto->get_contact_field_value('null', 'Coordinate_Tett...')
#9 {main}
